pekko-sample-distributed-data-scala/src/main/resources/application.conf

```diff
@@ -14,9 +14,9 @@ akka {
 
   cluster {
     seed-nodes = [
-      "akka://ClusterSystem@127.0.0.1:2551",
```

Comment:
could you use 7354 and 7355 as port numbers?

Reply:
7355 is Pekko default while 2552 is Akka default

Reply:
Updated